.github/workflows/draft-pdf.yml

```diff
@@ -0,0 +1,23 @@
+on: [push]
+
+jobs:
+  paper:
+    runs-on: ubuntu-latest
+    name: Paper Draft
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build draft PDF
+        uses: openjournals/openjournals-draft-action@master
+        with:
+          journal: joss
+          # This should be the path to the paper within your repo.
+          paper-path: doc/source/tracking/paper.md
+      - name: Upload
+        uses: actions/upload-artifact@v1
+        with:
+          name: paper
+          # This is the output path where Pandoc will write the compiled
+          # PDF. Note, this should be the same directory as the input
+          # paper.md
+          path: doc/source/tracking/paper.pdf
```

Comment:
GitHub Actions CI has been updated so the draft paper PDF is uploaded as an artifact. Get it by going to the 'Checks' tab at https://github.com/icesat2py/icepyx/pull/361/checks, then `.github/workflows/draft-pdf.yml`, and under 'Artifacts', there should be a `paper` link you can click to download a zipfile containing the PDF.

![image](https://user-images.githubusercontent.com/23487320/187531633-9c1fd27c-d9ff-466d-8dac-c55d36e556c5.png)

For example, the PDF for commit 45307dec37a6f0c8ca193d4102debfb78f0e383b is at https://github.com/icesat2py/icepyx/actions/runs/2958374263